docs/plugins/graphback-datasync.md

```diff
@@ -0,0 +1,225 @@
+---
```

Comment:
I think we need a separate chapter in website - datasync etc. 
The plugin itself is not enough.

Reply:
Agreed